site/app/views/LateDaysTableView.php

```diff
@@ -27,4 +27,21 @@ public function showLateTablePage(LateDays $late_days) {
             'grade_inquiry_enabled' => $this->core->getConfig()->isRegradeEnabled()
         ]);
     }
+
+    /**
+     * @param LateDays $late_days
+     * @return string
```

Comment:
Since there's just one type here, it would be change this to a native PHP type declaration.